src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -37,4 +50,15 @@ public void eliminateBigBusinesses() {
   public List<Listing> getBusinesses() {
     return businesses;
   }
+
+  private void addListingToBusinesses(PlacesSearchResult place) {
+    String name = place.name;
+    String formattedAddress = place.vicinity;
+    Geometry geometry = place.geometry;
+    MapLocation placeLocation = new MapLocation(geometry.location.lat, geometry.location.lng);
```

Comment:
nit: Prefer line length <= 80 characters.

Reply:
All lines are not the right length!